luxonis_ml/nn_archive/config_building_blocks/base_models/head.py

```diff
@@ -254,6 +255,30 @@ def validate_label_type(
         return value
 
 
+class HeadOBBDetectionYOLO(HeadObjectDetectionYOLO, ABC):
+    """Metadata for YOLO OBB head.
+
+    @type family: str
+    @ivar family: Decoding family.
+    @type outputs: C{OutputsOBBDetectionYOLO}
+    @ivar outputs: A configuration specifying which output names from the `outputs` block of the archive are fed into the head.
+    """
+
+    family: Literal["OBBDetectionYOLO"] = Field(..., description="Decoding family.")
+    outputs: OutputsOBBDetectionYOLO = Field(
+        description="A configuration specifying which output names from the `outputs` block of the archive are fed into the head."
+    )
+
```

Comment:
Do we need to store any additional information here? Like number of channels...?

I see later angles are defined as a separate output, so it might not be needed. Just thinking if this might be a useful information to have here?